scripts/executors/src/start.ts

```diff
@@ -195,15 +200,22 @@ async function getSelectedTarget(
   targetsDescription: Record<string, string>,
   omitTargets: string[],
   createDescription: TargetDescriptionFn,
+  preselectTarget: PreselectTargetFn,
 ): Promise<string> {
   const projectConfig = projects[selectedProject];
   const projectTargets = projectConfig.data.targets ?? {};
+  const preselectedTarget = preselectTarget(projectConfig);
 
   const availableTargets = Object.keys(projectTargets)
     .filter(targetName => {
       return omitTargets.includes(targetName) ? false : true;
```

Comment:
```suggestion
      return omitTargets.includes(targetName);
```

Reply:
aah yes, ty!

Reply:
hmm I prefer explicit code, this adds cognitive load